### ISSUE TITLE
Unable to get WallabyJS to work inside VS Code, with svelte kit app and jest test framework

### ISSUE THREAD

**Issue by abirtley** [wallabyjs/public#2953]
### Issue description or question

I created a default example Svelte Kit app, and have amended it to use Jest for testing instead of Playwright (using svelte-jester amongst other things).

On the command line, the jest tests work just fine.

Using WallabyJS inside VS Code, however, the tests do not run. The WallabyJS console indicates it is failing in the svelte-jester preprocess step:

```
​​[Error] Runtime error: Command failed: node --unhandled-rejections=strict --abort-on-uncaught-exception "./node_modules/svelte-jester/dist/preprocess.js"​​
​​[Error] internal/process/promises.js:194​​
​​[Error]         triggerUncaughtException(err, true /* fromPromise */);​​
​​[Error]         ^​​
​​[Error] TypeError [ERR_INVALID_ARG_TYPE]: The "chunk" argument must be of type string or an instance of Buffer. Received an instance of SyntaxError​​
​​[Error]     at validChunk (_stream_writable.js:281:10)​​
​​[Error]     at Socket.Writable.write (_stream_writable.js:316:21)​​
​​[Error]     at file://./node_modules/svelte-jester/dist/preprocess.js:21:32​​
​​[Error] Emitted 'error' event on Socket instance at:​​
​​[Error]     at errorOrDestroy (internal/streams/destroy.js:108:12)​​
​​[Error]     at validChunk (_stream_writable.js:284:5)​​
​​[Error]     at Socket.Writable.write (_stream_writable.js:316:21)​​
​​[Error]     at file://./node_modules/svelte-jester/dist/preprocess.js:21:32 {​​
​​[Error]   code: 'ERR_INVALID_ARG_TYPE'​​
​​[Error] }​​
​​[Error]  1: 0x1011f7a65 node::Abort() (.cold.1) [/Users/myusername/.nvm/versions/node/v12.22.5/bin/node]​​
```

It looks like the preprocessor is being invoked with node 12.22.5 instead of 14.17.5 like the rest of the project. However, if I `nvm use 12` on the command line and run `npx jest`, the tests still work, so I'm not sure if it's a simple node version issue.

A reproduction repo is available here: https://github.com/abirtley/svelte-wallaby-reproduction/

Thanks in advance for any insights you may have ☺️ 

### Wallaby diagnostics report

```
{
  editorVersion: '1.65.2',
  pluginVersion: '1.0.333',
  editorType: 'VSCode',
  osVersion: 'darwin 20.6.0',
  nodeVersion: 'v14.17.5',
  coreVersion: '1.0.1250',
  checksum: 'MDI0ODgzYWU0ZGU0OWE4MzlkZTdmZjNiMWZkNjMwZDIsMTY3ODMyMDAwMDAwMCww',
  config: {
    testFramework: { version: 'jest@24.8.0', configurator: 'jest@24.8.0', reporter: 'jest@24.8.0', starter: 'jest@24.8.0', configFile: './jest.config.js', autoDetected: true },
    diagnostics: {
      jest: {
        config: {
          configs: [
            {
              automock: false,
              cache: true,
              cacheDirectory: '/private/var/folders/kn/lrcbwj4x5vzfrhw3ln56yjnh0000gn/T/jest_dx',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '/node_modules/' ],
              cwd: '<homeDir>/Desktop/svelte-wallaby-reproduction',
              dependencyExtractor: undefined,
              detectLeaks: false,
              detectOpenHandles: false,
              displayName: undefined,
              errorOnDeprecated: false,
              extensionsToTreatAsEsm: [],
              extraGlobals: [],
              filter: undefined,
              forceCoverageMatch: [],
              globalSetup: undefined,
              globalTeardown: undefined,
              globals: {},
              haste: { computeSha1: false, enableSymlinks: false, forceNodeFilesystemAPI: false, throwOnModuleCollision: false },
              injectGlobals: true,
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [ 'js', 'ts', 'svelte' ],
              moduleLoader: undefined,
              moduleNameMapper: [ [ '^\\$lib/(.*)', '<homeDir>/Desktop/svelte-wallaby-reproduction/src/lib/$1' ] ],
              modulePathIgnorePatterns: [],
              modulePaths: undefined,
              name: 'f96be1e66a26c195cfe8865562347a5c',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: undefined,
              restoreMocks: false,
              rootDir: '<homeDir>/Desktop/svelte-wallaby-reproduction',
              roots: [ '<homeDir>/Desktop/svelte-wallaby-reproduction' ],
              runner: '<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/jest-runner/build/index.js',
              setupFiles: [],
              setupFilesAfterEnv: [ '<homeDir>/Desktop/svelte-wallaby-reproduction/jest-setup.ts' ],
              skipFilter: false,
              skipNodeResolution: undefined,
              slowTestThreshold: 5,
              snapshotFormat: undefined,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: '<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/jest-environment-jsdom/build/index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ '**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[tj]s?(x)' ],
              testPathIgnorePatterns: [ '/node_modules/' ],
              testRegex: [],
              testRunner: '<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/jest-circus/runner.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^.+\\.tsx?$', '<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/ts-jest/dist/index.js', {} ],
                [ '^.+\\.svelte$', '<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/svelte-jester/dist/transformer.cjs', { preprocess: true, enableSourcemap: true } ]
              ],
              transformIgnorePatterns: [ '/node_modules/', '\\.pnp\\.[^\\/]+$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            }
          ],
          globalConfig: {
            bail: 0,
            changedFilesWithAncestor: false,
            changedSince: undefined,
            collectCoverage: false,
            collectCoverageFrom: [],
            collectCoverageOnlyFrom: undefined,
            coverageDirectory: '<homeDir>/Desktop/svelte-wallaby-reproduction/coverage',
            coverageProvider: 'babel',
            coverageReporters: [ 'json', 'text', 'lcov', 'clover' ],
            coverageThreshold: undefined,
            detectLeaks: false,
            detectOpenHandles: false,
            errorOnDeprecated: false,
            expand: false,
            filter: undefined,
            findRelatedTests: false,
            forceExit: false,
            globalSetup: undefined,
            globalTeardown: undefined,
            json: false,
            lastCommit: false,
            listTests: false,
            logHeapUsage: false,
            maxConcurrency: 5,
            maxWorkers: 7,
            noSCM: undefined,
            noStackTrace: false,
            nonFlagArgs: undefined,
            notify: false,
            notifyMode: 'failure-change',
            onlyChanged: false,
            onlyFailures: false,
            outputFile: undefined,
            passWithNoTests: false,
            projects: [],
            replname: undefined,
            reporters: undefined,
            rootDir: '<homeDir>/Desktop/svelte-wallaby-reproduction',
            runTestsByPath: false,
            silent: undefined,
            skipFilter: false,
            snapshotFormat: undefined,
            testFailureExitCode: 1,
            testNamePattern: undefined,
            testPathPattern: '',
            testResultsProcessor: undefined,
            testSequencer: '<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/@jest/test-sequencer/build/index.js',
            testTimeout: undefined,
            updateSnapshot: 'new',
            useStderr: false,
            verbose: undefined,
            watch: false,
            watchAll: false,
            watchPlugins: undefined,
            watchman: true
          },
          hasDeprecationWarnings: false,
          wallaby: {
            roots: [],
            watchPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/', '/private/var/folders/kn/lrcbwj4x5vzfrhw3ln56yjnh0000gn/T/jest_dx', '\\./coverage' ],
            testPathIgnorePatterns: [ '/node_modules/', '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/', '/private/var/folders/kn/lrcbwj4x5vzfrhw3ln56yjnh0000gn/T/jest_dx', '\\./coverage' ],
            testMatch: [ '**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[tj]s?(x)' ],
            testRegex: []
          }
        }
      }
    },
    filesWithCoverageCalculated: [],
    filesWithNoCoverageCalculated: [],
    globalSetup: false,
    micromatch: true,
    files: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true },
      { pattern: '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/', regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\.\/git\/|\/\..+\//, ignore: true, trigger: true, load: true },
      {
        pattern: '/private/var/folders/kn/lrcbwj4x5vzfrhw3ln56yjnh0000gn/T/jest_dx',
        regexp: /\/private\/var\/folders\/kn\/lrcbwj4x5vzfrhw3ln56yjnh0000gn\/T\/jest_dx/,
        ignore: true,
        trigger: true,
        load: true
      },
      { pattern: '\\./coverage', regexp: /\.\/coverage/, ignore: true, trigger: true, load: true },
      { pattern: '**/**', ignore: false, trigger: true, load: true, order: 1 },
      { pattern: '**/__tests__/**/*.[jt]s?(x)', ignore: true, trigger: true, load: true },
      { pattern: '**/?(*.)+(spec|test).[tj]s?(x)', ignore: true, trigger: true, load: true }
    ],
    tests: [
      { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true, test: true },
      {
        pattern: '\\./dist/|\\./build/|\\./coverage/|\\./git/|/\\..+/',
        regexp: /\.\/dist\/|\.\/build\/|\.\/coverage\/|\.\/git\/|\/\..+\//,
        ignore: true,
        trigger: true,
        load: true,
        test: true
      },
      {
        pattern: '/private/var/folders/kn/lrcbwj4x5vzfrhw3ln56yjnh0000gn/T/jest_dx',
        regexp: /\/private\/var\/folders\/kn\/lrcbwj4x5vzfrhw3ln56yjnh0000gn\/T\/jest_dx/,
        ignore: true,
        trigger: true,
        load: true,
        test: true
      },
      { pattern: '\\./coverage', regexp: /\.\/coverage/, ignore: true, trigger: true, load: true, test: true },
      { pattern: '**/__tests__/**/*.[jt]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 2 },
      { pattern: '**/?(*.)+(spec|test).[tj]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 3 }
    ],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    compilers: {},
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    env: {
      type: 'node',
      params: {},
      runner: '<homeDir>/.nvm/versions/node/v14.17.5/bin/node',
      viewportSize: { width: 800, height: 600 },
      options: { width: 800, height: 600 },
      bundle: true
    },
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: true,
    configCode: "export default function () {\n  return {\n    autoDetect: true,\n    testFramework: {\n      configFile: './jest.config.js',\n    },\n  };\n};\n"
  },
  packageJSON: {
    dependencies: { '@fontsource/fira-mono': '^4.5.0', '@lukeed/uuid': '^2.0.0', cookie: '^0.4.1' },
    devDependencies: {
      '@sveltejs/adapter-auto': 'next',
      '@sveltejs/kit': 'next',
      '@testing-library/jest-dom': '^5.16.2',
      '@testing-library/svelte': '^3.1.0',
      '@types/cookie': '^0.4.1',
      '@types/jest': '^27.4.1',
      '@typescript-eslint/eslint-plugin': '^5.10.1',
      '@typescript-eslint/parser': '^5.10.1',
      eslint: '^7.32.0',
      'eslint-config-prettier': '^8.3.0',
      'eslint-plugin-svelte3': '^3.2.1',
      jest: '^27.5.1',
      prettier: '^2.5.1',
      'prettier-plugin-svelte': '^2.5.0',
      svelte: '^3.46.0',
      'svelte-check': '^2.2.6',
      'svelte-jester': '^2.3.2',
      'svelte-preprocess': '^4.10.1',
      'ts-jest': '^27.1.3',
      tslib: '^2.3.1',
      typescript: '~4.6.2'
    }
  },
  fs: { numberOfFiles: 32 },
  debug: [
    '2022-03-23T22:28:07.789Z angular/cli config Angular CLI not found.\n',
    '2022-03-23T22:28:07.882Z jest/config Detected Jest.\n',
    '2022-03-23T22:28:07.882Z jest/config Configured Jest.\n',
    '2022-03-23T22:28:07.884Z project Wallaby Node version: v14.17.5\n',
    '2022-03-23T22:28:07.884Z project Wallaby config: <homeDir>/Desktop/svelte-wallaby-reproduction/wallaby.js\n',
    '2022-03-23T22:28:07.930Z project File cache: <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.333/projects/06974f38dfafe349\n',
    '2022-03-23T22:28:08.053Z uiService Listening port 51235\n',
    '2022-03-23T22:28:08.066Z workers Parallelism for initial run: 6, for regular run: 3\n',
    '2022-03-23T22:28:08.066Z workers Starting run worker instance #0\n',
    '2022-03-23T22:28:08.067Z workers Starting run worker instance #1\n',
    '2022-03-23T22:28:08.067Z workers Starting run worker instance #2\n',
    '2022-03-23T22:28:08.067Z workers Starting run worker instance #3\n',
    '2022-03-23T22:28:08.067Z workers Starting run worker instance #4\n',
    '2022-03-23T22:28:08.067Z workers Starting run worker instance #5\n',
    '2022-03-23T22:28:08.067Z workers Web server is listening at 59578\n',
    '2022-03-23T22:28:08.100Z project File cache requires some updates, waiting required files from IDE\n',
    '2022-03-23T22:28:08.166Z project Stopping process pool\n',
    '2022-03-23T22:28:08.173Z project Test run started; run priority: 3\n',
    '2022-03-23T22:28:08.174Z project Running all tests\n',
    '2022-03-23T22:28:08.175Z workers Starting test run, priority: 3\n',
    '2022-03-23T22:28:08.176Z workers Distributing tests between 6 workers\n',
    '2022-03-23T22:28:08.176Z workers Running tests in parallel\n',
    '2022-03-23T22:28:08.176Z nodeRunner Starting sandbox [worker #0, session #r8ucb]\n',
    '2022-03-23T22:28:08.176Z nodeRunner Starting sandbox [worker #1, session #rcp6l]\n',
    '2022-03-23T22:28:08.176Z nodeRunner Preparing sandbox [worker #0, session #r8ucb]\n',
    '2022-03-23T22:28:08.176Z nodeRunner Preparing sandbox [worker #1, session #rcp6l]\n',
    '2022-03-23T22:28:08.252Z workers Started run worker instance (delayed) #0\n',
    '2022-03-23T22:28:08.253Z nodeRunner Prepared sandbox [worker #0, session #r8ucb]\n',
    '2022-03-23T22:28:08.253Z workers [worker #0, session #r8ucb] Running tests in sandbox\n',
    '2022-03-23T22:28:08.273Z workers Started run worker instance (delayed) #1\n',
    '2022-03-23T22:28:08.273Z nodeRunner Prepared sandbox [worker #1, session #rcp6l]\n',
    '2022-03-23T22:28:08.273Z workers [worker #1, session #rcp6l] Running tests in sandbox\n',
    '2022-03-23T22:28:08.277Z workers Started run worker instance (delayed) #2\n',
    '2022-03-23T22:28:08.296Z workers Started run worker instance (delayed) #3\n',
    '2022-03-23T22:28:08.314Z workers Started run worker instance (delayed) #5\n',
    '2022-03-23T22:28:08.322Z workers Started run worker instance (delayed) #4\n',
    '2022-03-23T22:28:11.267Z workers Scheduling Jest Test Run (rcp6l): 2022-03-23T22:28:08.697Z\n',
    '2022-03-23T22:28:11.270Z workers Sandbox (active) [rcp6l] error: Command failed: node --unhandled-rejections=strict --abort-on-uncaught-exception "<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/svelte-jester/dist/preprocess.js"\n' +
      'internal/process/promises.js:194\n' +
      '        triggerUncaughtException(err, true /* fromPromise */);\n' +
      '        ^\n' +
      '\n' +
      'TypeError [ERR_INVALID_ARG_TYPE]: The "chunk" argument must be of type string or an instance of Buffer. Received an instance of SyntaxError\n' +
      '    at validChunk (_stream_writable.js:281:10)\n' +
      '    at Socket.Writable.write (_stream_writable.js:316:21)\n' +
      '    at file://<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/svelte-jester/dist/preprocess.js:21:32\n' +
      "Emitted 'error' event on Socket instance at:\n" +
      '    at errorOrDestroy (internal/streams/destroy.js:108:12)\n' +
      '    at validChunk (_stream_writable.js:284:5)\n' +
      '    at Socket.Writable.write (_stream_writable.js:316:21)\n' +
      '    at file://<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/svelte-jester/dist/preprocess.js:21:32 {\n' +
      "  code: 'ERR_INVALID_ARG_TYPE'\n" +
      '}\n' +
      ' 1: 0x1011f7a65 node::Abort() (.cold.1) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 2: 0x1000a5fd9 node::Abort() [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 3: 0x1000a75ea node::errors::TriggerUncaughtException(v8::FunctionCallbackInfo<v8::Value> const&) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 4: 0x10025b5d8 v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 5: 0x10025ab99 v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 6: 0x10025a302 v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 7: 0x100a04199 Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_BuiltinExit [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 8: 0x100989924 Builtins_InterpreterEntryTrampoline [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 9: 0x100989924 Builtins_InterpreterEntryTrampoline [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      '10: 0x100986e9d Builtins_JSEntryTrampoline [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      '\n',
    '2022-03-23T22:28:11.272Z workers [rcp6l] Run 0 test(s), skipped 0 test(s)\n',
    '2022-03-23T22:28:11.273Z workers Jest Test Run Complete (rcp6l): 2022-03-23T22:28:11.266Z\n',
    '2022-03-23T22:28:11.274Z workers [rcp6l] Sandbox is responsive, closing it\n',
    '2022-03-23T22:28:11.285Z workers Scheduling Jest Test Run (r8ucb): 2022-03-23T22:28:08.698Z\n',
    '2022-03-23T22:28:11.288Z workers Sandbox (active) [r8ucb] error: Command failed: node --unhandled-rejections=strict --abort-on-uncaught-exception "<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/svelte-jester/dist/preprocess.js"\n' +
      'internal/process/promises.js:194\n' +
      '        triggerUncaughtException(err, true /* fromPromise */);\n' +
      '        ^\n' +
      '\n' +
      'TypeError [ERR_INVALID_ARG_TYPE]: The "chunk" argument must be of type string or an instance of Buffer. Received an instance of SyntaxError\n' +
      '    at validChunk (_stream_writable.js:281:10)\n' +
      '    at Socket.Writable.write (_stream_writable.js:316:21)\n' +
      '    at file://<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/svelte-jester/dist/preprocess.js:21:32\n' +
      "Emitted 'error' event on Socket instance at:\n" +
      '    at errorOrDestroy (internal/streams/destroy.js:108:12)\n' +
      '    at validChunk (_stream_writable.js:284:5)\n' +
      '    at Socket.Writable.write (_stream_writable.js:316:21)\n' +
      '    at file://<homeDir>/Desktop/svelte-wallaby-reproduction/node_modules/svelte-jester/dist/preprocess.js:21:32 {\n' +
      "  code: 'ERR_INVALID_ARG_TYPE'\n" +
      '}\n' +
      ' 1: 0x1011f7a65 node::Abort() (.cold.1) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 2: 0x1000a5fd9 node::Abort() [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 3: 0x1000a75ea node::errors::TriggerUncaughtException(v8::FunctionCallbackInfo<v8::Value> const&) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 4: 0x10025b5d8 v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 5: 0x10025ab99 v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 6: 0x10025a302 v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 7: 0x100a04199 Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_BuiltinExit [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 8: 0x100989924 Builtins_InterpreterEntryTrampoline [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      ' 9: 0x100989924 Builtins_InterpreterEntryTrampoline [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      '10: 0x100986e9d Builtins_JSEntryTrampoline [<homeDir>/.nvm/versions/node/v12.22.5/bin/node]\n' +
      '\n',
    '2022-03-23T22:28:11.289Z workers [r8ucb] Run 0 test(s), skipped 0 test(s)\n',
    '2022-03-23T22:28:11.289Z workers Jest Test Run Complete (r8ucb): 2022-03-23T22:28:11.284Z\n',
    '2022-03-23T22:28:11.290Z workers [r8ucb] Sandbox is responsive, closing it\n',
    '2022-03-23T22:28:11.291Z workers Merging parallel test run results\n',
    '2022-03-23T22:28:11.291Z project Test run finished\n',
    '2022-03-23T22:28:11.292Z project Processed console.log entries\n',
    '2022-03-23T22:28:11.292Z project Processed loading sequences\n',
    '2022-03-23T22:28:11.292Z project Processed executed tests\n',
    '2022-03-23T22:28:11.292Z project Processed code coverage\n',
    '2022-03-23T22:28:11.300Z project Test run result processed and sent to IDE\n'
  ]
}
```

**Comment by smcenlly** [comment]
Would you be able to try the following:

1. Switch to use node 14 `nvm use 14`
2. Reboot your machine
3. Start Wallaby

I'm wondering if you have somehow switched node versions at some point and Wallaby was started with 14 (e.g. from VS Code) but your system is using `node 12`. It seems like for your machine/environment, when you are using Wallaby, [this code](https://github.com/mihar-22/svelte-jester/blob/a871addeb1176ca217a2d7e5abef395f26ec06d5/src/transformer.js#L47) in svelter is resolving `node 12` instead of `node 14` which is causing problems.

If that doesn't help, please let us know.

**Comment by abirtley** [comment]
🤦 turn it off and on again - why didn't I think of that?

I followed those steps, adding in an additional step 1.a: `nvm alias default 14` just in case

Everything now works in Wallaby.

(Although for anyone following this trail who then runs into the [dreaded import.meta not allowed issue](https://github.com/kulshekhar/ts-jest/issues/1174), I worked around it [using a Jest mock as suggested here](https://github.com/kulshekhar/ts-jest/issues/1174#issuecomment-939285099).)

Thank you so much!
